apps/trading/client-pages/markets/markets-page.tsx

```diff
@@ -125,78 +166,355 @@ export const MarketsPage = () => {
+                      if (checked) {
+                        setMarketStates(uniq([...marketStates, marketState]));
+                      } else {
+                        setMarketStates(
+                          marketStates.filter((s) => s !== marketState)
+                        );
+                      }
+                    }}
+                    key={key}
+                  >
+                    {marketStateFilterOptions[marketState]}
+                  </MultiSelectOption>
+                );
+              })}
+            </MultiSelect>
+          </div>
+          <div>
+            <MultiSelect placeholder="Assets" trigger={assetTrigger}>
+              {assetFilterOptions.map((asset) => {
+                const chainName = getChainName(
+                  asset.source.__typename === 'ERC20'
+                    ? Number(asset.source.chainId)
+                    : undefined
+                );
+                const isChecked = assets.includes(asset.id);
+                return (
+                  <MultiSelectOption
+                    checked={isChecked}
+                    onCheckedChange={(checked) => {
+                      if (checked) {
+                        setAssets(uniq([...assets, asset.id]));
+                      } else {
+                        setAssets(assets.filter((a) => a !== asset.id));
+                      }
+                    }}
+                    key={asset.id}
+                  >
+                    <span>{asset.symbol}</span>{' '}
+                    <span className="text-xs">({chainName})</span>
+                  </MultiSelectOption>
+                );
+              })}
+            </MultiSelect>
+          </div>
+          {/** MARKET NAME FILTER */}
+          <div className="w-2/3">
+            <TradingInput
+              prependIconName={VegaIconNames.SEARCH}
+              prependIconDescription={t('Search by market')}
+              placeholder={t('Search by market')}
+              className="text-sm border !placeholder:text-secondary"
+              onChange={(ev) => {
+                const value = trim(ev.target.value);
+                setSearchTerm(value);
+              }}
+              value={searchTerm}
+              defaultValue={''}
+            />
+          </div>
+        </div>
       </div>
 
       <div className="h-full my-1 border rounded border-default">
-        {activeTab === 'open-markets' && (
-          <ErrorBoundary feature="markets-open">
-            <OpenMarkets data={activeMarkets} error={error} />
-          </ErrorBoundary>
-        )}
-        {activeTab === 'proposed-markets' && (
-          <ErrorBoundary feature="markets-proposed">
-            <Proposed />
-          </ErrorBoundary>
-        )}
-        {activeTab === 'closed-markets' && (
-          <ErrorBoundary feature="markets-closed">
-            <Closed />
-          </ErrorBoundary>
-        )}
+        <ErrorBoundary feature="all-markets">
+          <OpenMarkets
```

Comment:
We should rename this to or remove one level of components here as this is no longer a component for just open markets